examples/preprocessing/plot_eog_artifact_histogram.py

```diff
@@ -41,7 +41,7 @@
 
 # Get the stim channel data
 pick_ch = mne.pick_channels(epochs.ch_names, ['STI 014'])[0]
-data = epochs.get_data()[:, pick_ch, :].astype(int)
+data = epochs.get_data(picks=pick_ch).astype(int)
```

Comment:
dimensionality change here [broke circleci](https://circleci.com/gh/mne-tools/mne-python/12491)

Reply:
(it has a singleton dimension where it did not before)